src/components/GuideCardContainer.astro

```diff
@@ -0,0 +1,70 @@
+---
+import { PAGES } from "../utils/constants.js";
+import GuideCard from "../components/GuideCard.astro";
+---
+
+<section>
+  <h2 class="mb-16 text-6xl">Guides</h2>
+  <div class="grid grid-cols-1 gap-12 md:grid-cols-2 lg:grid-cols-3">
+    <GuideCard
+      icon="tabler:old"
```

Comment:
Next steps are to rewrite this section to iterate through the PAGES object imported above and to create the Guide card items

Reply:
The issue for that is here -> https://github.com/CareerCatalyst/Career-Catalyst/issues/15